checker/tests/nullness/TestPolyAll2.java

```diff
@@ -12,7 +12,7 @@ public static boolean noDuplicates1(@PolyAll @NonNull @UnknownKeyFor String[] a)
         return true;
     }
 
-    public static boolean noDuplicates2(@PolyAll @Nullable String[] a) {
+    public static boolean noDuplicates2(@PolyAll @UnknownKeyFor @Nullable String[] a) {
```

Comment:
I don't understand why this change is required. Isn't `@UnknownKeyFor` the default?

Reply:
Without `@UnknownKeyFor`, the `@PolyAll` is that annotation in the Key For hierarchy.  Line 17, `a[0] = null` issue an error without this annotation.

Reply:
Ah, true. Let's hope not many people use PolyAll, otherwise they will get many unexpected warnings.